docs/ImportantPythonFunctions.md

```diff
@@ -0,0 +1,180 @@
+# Important Python Functions in ONNX
+
+## Loading an ONNX Model
+```python
+import onnx
+
+onnx_model = onnx.load('path/to/the/model')
+```
+Runnable IPython notebooks:
+- [load_model.ipynb](https://github.com/onnx/onnx/tree/master/onnx/examples/load_model.ipynb)
+
+## Saving an ONNX Model
+```python
+import onnx
+
+onnx_model = ... # Your model in memory
+
+# Save the ONNX model
```

Comment:
Can be done in a separate diff: We should wrap this into a helper function (say `save_model`), especially soon we will have support for storing tensor values in separate files.

Reply:
Good point. Working on it, will rebase this diff on that one.

Reply:
https://github.com/onnx/onnx/pull/692